File: src/main/java/org/jukeboxmc/potion/FireResistanceEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class FireResistanceEffect extends Effect {

    @Override
    public int getId() {
        return 12;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.FIRE_RESISTANCE;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 228, 154, 58 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/FireResistanceEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class FireResistanceEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 12
    }

    override fun getEffectType(): EffectType {
        return EffectType.FIRE_RESISTANCE
    }

    override fun getEffectColor(): Color {
        return Color(228, 154, 58)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}